app/config/reporting_api_version

```diff
@@ -1 +1 @@
-master
+i2385
```

Comment:
Need to change this back to master after i2385 has been merged (montagu API allowing access to changelogs for readers)